src/wrappedmap.jl

```diff
@@ -17,6 +17,22 @@ function WrappedMap{T}(lmap::Union{AbstractMatrix, LinearMap};
     WrappedMap{T, typeof(lmap)}(lmap, issymmetric, ishermitian, isposdef)
 end
 
+const MatrixMap{T} = WrappedMap{T,<:AbstractMatrix}
+
+LinearAlgebra.transpose(A::MatrixMap{T}) where {T} =
+    WrappedMap{T}(transpose(A.lmap); issymmetric=A._issymmetric, ishermitian=A._ishermitian, isposdef=A._isposdef)
+LinearAlgebra.adjoint(A::MatrixMap{T}) where {T} =
+    WrappedMap{T}(adjoint(A.lmap); issymmetric=A._issymmetric, ishermitian=A._ishermitian, isposdef=A._isposdef)
+
+Base.:(==)(A::MatrixMap, B::MatrixMap) = (eltype(A) == eltype(B) && A.lmap == B.lmap)
```

Comment:
I would be inclined to also check the specifiers `_issymmetric` etc here. If a user went through the effort of creating one `WrappedMap` where he set `issymmetric` to true, and another one with the same matrix where he set `issymmetric` to false, he probably wants them to be different. (Not that I see this happening in any realistic use case).